packages/react/src/index.ts

```diff
@@ -8,7 +8,9 @@ export { createReduxEnhancer } from './redux';
 export { reactRouterV3Instrumentation } from './reactrouterv3';
 export { reactRouterV4Instrumentation, reactRouterV5Instrumentation, withSentryRouting } from './reactrouter';
 export {
+  // eslint-disable-next-line deprecation/deprecation
   reactRouterV6Instrumentation,
+  browserTracingReactRouterV6Integration,
```

Comment:
Same as in https://github.com/getsentry/sentry-javascript/pull/10489, I think it would be cleaner to call it `reactRouterV6BrowserTracingIntegration`.